manifests/custom-resource-definitions/reportdatasource.crd.yaml

```diff
@@ -3,8 +3,9 @@ kind: CustomResourceDefinition
 metadata:
   name: reportdatasources.metering.openshift.io
   annotations:
-    catalog.app.coreos.com/displayName: "Metering data source"
-    catalog.app.coreos.com/description: "A resource describing a source of data for usage by Report Generation Queries"
+    catalog.app.coreos.com/displayName: Metering Data Source
+    catalog.app.coreos.com/description: A resource describing a source of data for usage by Report Generation Queries.
+    catalog.app.coreos.com/weight: 4
 spec:
```

Comment:
Do these need to be strings?
catalog.app.coreos.com/displayName
catalog.app.coreos.com/description

Reply:
Those are strings, yaml doesn't require quotes. That being said; the `weight` probably does need quotes.